migrations/versions/6e9328b6b275_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
```

Comment:
What does this revision do? It appears to point to two heads. Perhaps this interfaces with the PR @zschira wrote up about revising the two-headed nightly build?

Reply:
This was the result of running `alembic merge heads` with Zach's PR. I tried to consolidate them, but it didn't work.